pycloudlib/instance.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, key_pair):
         self.port = '22'
         self.username = 'ubuntu'
         self.connect_timeout = 60
+        self.banner_timeout = 60
```

Comment:
While it would be nice to provide config overrides for this timeout, I think this is something that needs to be addressed separately as all these instance attrs should be configurable at some point in the future